content/en/platform/corda/5.0/developing-applications/api/api-crypto.md

```diff
@@ -10,9 +10,9 @@ menu:
 section_menu: corda5
 ---
 # net.corda.v5.crypto
-The `corda-crypto` module is one of several modules of the `Corda Crypto API`. The module defines low-level services that can be used to extend functionality of the Corda Crypto Library by implementing them in a CPK. 
+The `crypto` module defines low-level services that can be used to extend functionality of the Corda Crypto Library by implementing them in a CPK. 
```

Comment:
It's actually `crypto-extensions` module which is meant to extend crypto capabilities(?) for CorDapps (for example a custom digest algorithm which mentioned below).

> low-level services

I believe is too vague here. Perhaps _crypto capabilities_?